react/src/views/Statistics/components/PeopleByAgeChart.tsx

```diff
@@ -0,0 +1,77 @@
+import BarChartCenterAxis, { IBarChartCenterAxis } from "components/graphs/BarChartCenterAxis";
+import { random, uniq } from "lodash";
+
+const dataCube = {
+  factTable: [
+    { gender: "M", age: 1, created: "2021-06-22", people: 3 },
```

Comment:
@MaikNeubert [this post](https://github.com/boxwise/boxtribute/issues/888#issuecomment-1670889773) shows the data format provide by the BE. 
1. `gender` will be one of Male/Female/Diverse
2. `created` will be `createdOn`
3. `people` will be `count`
(1.) cannot be changed since it's dictated by the GraphQL enum HumanGender. (2.) can be changed but would be inconsistent (datetime fields in our GraphQL schema always have suffix `On`). (3.) can be changed :slightly_smiling_face:

Reply:
fields on the frontend where updated to match the BE